File: backend/src/main/java/com/team05/petmeeting/domain/comment/dto/FeedCommentListRes.java
package com.team05.petmeeting.domain.comment.dto;

import java.util.List;

public record FeedCommentListRes (
        List<FeedCommentRes> comments,
        int totalCount
){
    public static FeedCommentListRes from(List<FeedCommentRes> comments){
        return new FeedCommentListRes(comments, comments.size());
    }
}

File: backend/src/main/java/com/team05/petmeeting/domain/comment/dto/FeedCommentListRes.kt
package com.team05.petmeeting.domain.comment.dto

data class FeedCommentListRes(
    val comments: List<FeedCommentRes>,
    val totalCount: Int
) {
    companion object {
        @JvmStatic
        fun from(comments: List<FeedCommentRes>): FeedCommentListRes {
            return FeedCommentListRes(comments, comments.size)
        }
    }
}
